Makefile

```
@@ -67,10 +68,22 @@ check-format:
 	npx prettier --check "**/*" --ignore-unknown
 
 # ref https://superuser.com/a/1170997
-.PHONY: bundle-python
-bundle-python:
+.PHONY: bundle-python-mac
+bundle-python-mac:
 	ditto -c -k --sequesterRsrc --keepParent ./mockup_package/mockup ./public/mockup.zip
 
+.PHONY: bundle-python-linux
+bundle-python-linux:
+	mkdir -p tmp/mockup
+	cp -r mockup_package/mockup/ tmp/mockup/
+	cd tmp && \
+	if [ -d "mockup" ]; then \
+	  zip -r ../public/mockup mockup; \
+	else \
+	  echo "Error: Directory 'mockup' does not exist."; \
+	  exit 1; \
+	fi
+
```

Why need cd into `tmp`?

```suggestion
	if [ -d "tmp/mockup" ]; then \
	  zip -r public/mockup tmp/mockup; \
	else \
	  echo "Error: Directory 'mockup' does not exist in directory 'tmp'."; \
	  exit 1; \
	fi

```

---

@pkong-ds  if we use `zip -r public/mockup tmp/mockup;` the unzip file will be
```
tmp/
    mockup/
```

---

okay, let's keep it then